### ISSUE TITLE
Remove the outline from the custom buttons on the dropdown

### ISSUE THREAD

**Issue by ravenac95** [conorgil/2fa-notifier#28]
@conorgil @designedbinary you should get rid of the blue outline: 

<img width="278" alt="screen shot 2018-05-04 at 10 48 35 pm" src="https://user-images.githubusercontent.com/862590/39660125-58a82800-4fed-11e8-9aeb-021084461a4e.png">

**Comment by ravenac95** [comment]
Here's the solution ```
.someclass:focus {
  outline: 0;
}

// can't remember if you need this next section
.someclass:active {
  outline: 0; 
}
```

**Comment by designedbinary** [comment]
@ravenac95, Does this show up for you by default?

**Comment by designedbinary** [comment]
@conorgil, I don't think this should be part of the "initial release." From an accessibility standpoint, we'll need some focus indicators. Its just a matter for making them better incorporate with the overall design. 

So, I want to address this issue, but don't think its worth blocking release.

**Comment by conorgil** [comment]
@designedbinary Agreed. This is a minor bug at best. There are more important things to work on right now.

**Comment by conorgil** [comment]
@ravenac95 I don't think we addressed this specifically, but I also don't see the blue box around the menu anymore. Does it still show for you? If no, can you close this issue :)
